### PR TITLE
Create chains.json

### PR DESCRIPTION
**Pull request by chronobit-dev**

### CODE REVIEW

File: chronobit/chains.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../chains.schema.json",
+  "name": "ChronoBit|Enterprise",
+  "chains": [
+    {
+      "name": "sentinel",
+      "address": "sentvalcons136f8p02t8903trmdt8ekrs3l4r0ursas7wj8vx",
```
Comment (tombeynon) [inline chronobit/chains.json:7]: I'll merge but this is usually sentvaloper1.. address rather than sentvalcons. Might need changing

**Review by tombeynon — APPROVED** [review]

File: chronobit/chains.json (hunk 1 above)
Comment (chronobit-dev) [reply chronobit/chains.json:7]: Let me double check and confirm

Comment (chronobit-dev) [reply chronobit/chains.json:7]: Can you update it or do I have to do another pull request? I have the valoper address.

Comment (tombeynon) [reply chronobit/chains.json:7]: I'm not at a computer right now so would be quicker to submit another PR

**Pull request merged**